pytorch_lightning/accelerators/gpu.py

```diff
@@ -51,11 +49,3 @@ def set_nvidia_flags(local_rank: int) -> None:
         all_gpu_ids = ",".join([str(x) for x in range(torch.cuda.device_count())])
         devices = os.getenv("CUDA_VISIBLE_DEVICES", all_gpu_ids)
         _log.info(f"LOCAL_RANK: {local_rank} - CUDA_VISIBLE_DEVICES: [{devices}]")
-
-    def to_device(self, step_kwargs: Dict[str, Union[Any, int]]) -> Dict[str, Union[Any, int]]:
-        # no need to transfer batch to device in DP mode
-        # TODO: Add support to allow batch transfer to device in Lightning for DP mode.
```

Comment:
Is this limitation removed?

Do you need to also update `ConfigValidator.__verify_dp_batch_transfer_support`?

Reply:
no, it is not. 

I have moved this comment and the check to the accelerator class where we handle the batch transfer. 
I removed the TODO because it is so extremely unrealistic that we can ever add support for this, unless pytorch adds some hooks to DataParallel that we can use for a custom scatter and gather.